build/images/deps/suricata-version

```diff
@@ -1 +1 @@
-6.0
+7.0
```

Comment:
I think this should be done in a separate, dedicated PR.
The PR description does not mention why this is necessary for this feature.